target/linux/ath79/image/lzma-loader/src/board.c

```diff
@@ -182,34 +182,40 @@ static inline void mr18_init(void)
 static inline void mr18_init(void) { }
 #endif
 
-#ifdef CONFIG_BOARD_HUAWEI_AP5030DN
-static inline void ap5030dn_init(void)
+#if defined(CONFIG_BOARD_HUAWEI_AP5030DN) || defined(CONFIG_BOARD_HUAWEI_AP6010DN)
+static inline void huawei_apXXXXdn_init(void)
```

Comment:
Rename to huawei_ap_init, we dont have others from Huawei which do need this hack yet.

Reply:
done